<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form2</class>
 <widget class="QMainWindow" name="form2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>m</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QSpinBox" name="spinBox_3">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>-9999999</number>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>результат:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinBox"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>n</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Создать таблицу</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="4">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox_2"/>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Результат</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>sum</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mult</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBox_4"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>k</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
